dGame/dComponents/TriggerComponent.cpp

```diff
@@ -0,0 +1,214 @@
+#include "TriggerComponent.h"
+
+#include "dZoneManager.h"
+#include "LUTriggers.h"
+#include "eTriggerCommandType.h"
+
+#include "MissionComponent.h"
+#include "PhantomPhysicsComponent.h"
+
+#include "CDMissionTasksTable.h"
```

Comment:
Any reason behind the spacing here for all of our own source files?  Generally you'll want to space out things like thirdparty, C/C++ library headers and others but the splitting of our own is a bit odd to see.

Reply:
just me being weird